### DISCUSSION TITLE
Add Temporal Bone/Mastoid Window Level Preset

### DISCUSSION THREAD

**Discussion by r4fh4f** [nroduit/Weasis#613] [Q&A]
I would like to suggest the inclusion of a default window level preset specifically for temporal bone and mastoid studies, with approximate values of 4000 / 700.
This preset would be particularly helpful for efficient visualization of temporal bone structures and would save time during routine image analysis. Currently, these values need to be adjusted manually for each examination.
Thank you for considering this feature request. Your work on Weasis is greatly appreciated by the medical imaging community.
Best regards,

**Comment by nroduit** [comment]
See https://github.com/nroduit/Weasis/issues/249#issuecomment-1082754067
